docs/.vitepress/vitepress/components/common/vp-theme-toggler.vue

```diff
@@ -1,50 +1,25 @@
 <script setup lang="ts">
+import { isDark } from '../../composables/dark'
 import DarkIcon from '../icons/dark.vue'
 import LightIcon from '../icons/light.vue'
-import VPSwitch from './vp-switch.vue'
 </script>
 
 <template>
-  <VPSwitch>
-    <ElIcon :size="13">
-      <DarkIcon class="dark-icon" />
-      <LightIcon class="light-icon" />
-    </ElIcon>
-  </VPSwitch>
+  <el-switch
+    :model-value="isDark"
```

Comment:
cuz u use isDark which refs to the native browser API so u should use ClientOnly to wrap it